src/javascript/static/pages/tnc.js

```diff
@@ -1,6 +1,7 @@
 const tabListener = require('@binary-com/binary-style').tabListener;
 const localize    = require('../../_common/localize').localize;
 const TNCApproval = require('../../app/pages/user/tnc_approval');
+const LocalStore       = require('../../_common/storage').LocalStore;
```

Comment:
extra whitespace before `=`

Reply:
also this should be alphabetical by pathname

Reply:
corrected